modules/bibdocfile/lib/bibdocfile.py

```diff
@@ -1657,7 +1657,7 @@ def create_new_document(doc_type="Main", rec_links=None):
         except Exception, e:
             run_sql('DELETE FROM bibdoc WHERE id=%s', (doc_id, ))
             register_exception(alert_admin=True)
-            raise InvenioBibDocFileError, e
```

Comment:
true, but you don't always read the commit messages when you are browsing the code :)
In other cases, we catch the exceptions somewhere and in this specific one we really want it to fail, so maybe a comment could help avoiding confusion.